plugins/source/aws/resources/services/budgets/budgets.go

```diff
@@ -0,0 +1,44 @@
+package budgets
+
+import (
+	"context"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/service/budgets"
+	"github.com/aws/aws-sdk-go-v2/service/budgets/types"
+	"github.com/cloudquery/cloudquery/plugins/source/aws/client"
+	"github.com/cloudquery/plugin-sdk/v4/schema"
+	"github.com/cloudquery/plugin-sdk/v4/transformers"
+)
+
+func Budgets() *schema.Table {
+	tableName := "aws_budgets"
```

Comment:
Why is this not `aws_budget_budgets`? In GCP we have `gcp_workflows_workflows`. I think it is probably better to be consistent in our naming rather than try and make it the most readable/understandable

Reply:
We already have `aws_organizations` & `aws_regions` tables.
Besides, I feel like having `aws_%svc%_%resource%` is an artifact of codegen, so we might just go with better names for the new resources.

Reply:
I would be in favor in our next breaking change to add the service name to those tables that only have a resource name...If we are going to move away from the `aws_%svc%_%resource%` naming convention we should do it explicitly with the entire teams buy in.